app/controllers/users_controller.rb

```diff
@@ -0,0 +1,32 @@
+class UsersController < ApplicationController
+  def show
+    user = User.find_by id: params[:id]
```

Comment:
```suggestion
    @user = User.find_by id: params[:id]
    return if @user
    
    flash[:danger] = t "flash.not_found_user"
    redirect_to root_path
```

Tạm thời viết vậy, sau này sẽ có cmt thêm